src/main/scala/akka/persistence/cassandra/CassandraMetricsRegistry.scala

```diff
@@ -0,0 +1,36 @@
+/*
+ * Copyright (C) 2016 Typesafe Inc. <http://www.typesafe.com>
+ */
+package akka.persistence.cassandra
+
+import akka.actor._
+import com.codahale.metrics.MetricRegistry
+import scala.concurrent.{ ExecutionContext, Future }
+import scala.collection.convert.wrapAsScala._
+import ExecutionContext.Implicits.global
+
+/**
+ * Retrieves Cassandra metrics registry for an actor system
+ */
+class CassandraMetricsRegistryImpl extends Extension {
```

Comment:
I normally name this class the same as the companion factory, i.e. `CassandraMetricsRegistry`